src/Makefile.am

```
@@ -691,28 +720,18 @@ libzcash_a_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
 if BUILD_BITCOIN_LIBS
 include_HEADERS = script/zcash_script.h
 libzcash_script_la_SOURCES = \
-  crypto/hmac_sha512.cpp \
-  crypto/ripemd160.cpp \
-  crypto/sha1.cpp \
-  crypto/sha256.cpp \
-  crypto/sha512.cpp \
-  hash.cpp \
-  primitives/transaction.cpp \
-  primitives/tx_version_info.cpp \
-  pubkey.cpp \
-  script/zcash_script.cpp \
-  script/interpreter.cpp \
-  script/script.cpp \
-  uint256.cpp \
-  util/strencodings.cpp \
-  zip317.cpp
+  $(crypto_libbitcoin_crypto_a_SOURCES) \
+  $(libbitcoin_script_a_SOURCES) \
+  $(libcxxbridge_a_SOURCES) \
+  support/cleanse.cpp \
+  uint256.cpp
 
 if GLIBC_BACK_COMPAT
   libzcash_script_la_SOURCES += compat/glibc_compat.cpp
 endif
 
 libzcash_script_la_LDFLAGS = $(AM_LDFLAGS) -no-undefined $(RELDFLAGS)
```

```suggestion
libzcash_script_la_LDFLAGS = $(AM_LDFLAGS) --no-undefined --error-unresolved-symbols $(RELDFLAGS)
```
See the ld.lld(1) man page. Non-blocking.